scripts/runTests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./node_modules/.bin/protractor okta-oidc-tck/e2e-tests/custom-login/conf.js
+./node_modules/.bin/protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js
```

Comment:
How does this script declare TEST_TYPE=implicit for the TCK ?

Reply:
Right. So, this is the issue I was talking about in the morning where I couldn't get the resource server working on windows.
Currently this doesn't set the `TEST_TYTPE=implicit` -> So we are not testing the resource server.
I will create a separate task to address this issue.

Reply:
@robertjd - Found out the issue with resource server not working on windows (mvn command) 
Will submit a patch with the fix to start resource server with `TEST_TYPE=implicit`